examples/google-analytics/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+
+/.cache
+/build
+/public/build
+.env
```

Comment:
```suggestion
.env

/package-lock.json
/yarn.lock 
```

Reply:
Not necessary, as they're already ignored in the `examples` folder itself